packages/integration-tests/src/entities/AuthorCodegen.ts

```diff
@@ -1,51 +1,51 @@
 import {
+  Flavor,
+  ValueFilter,
+  ValueGraphQLFilter,
+  OrderBy,
+  ConfigApi,
   BaseEntity,
-  BooleanFilter,
-  BooleanGraphQLFilter,
+  EntityManager,
+  setOpts,
+  PartialOrNull,
+  OptsOf,
   Changes,
-  Collection,
-  ConfigApi,
+  newChangesProxy,
+  Lens,
+  loadLens,
+  LoadHint,
+  Loaded,
+  getEm,
+  BooleanFilter,
   EntityFilter,
-  EntityGraphQLFilter,
-  EntityManager,
   FilterOf,
-  Flavor,
-  getEm,
+  BooleanGraphQLFilter,
+  EntityGraphQLFilter,
   GraphQLFilterOf,
+  newRequiredRule,
+  Collection,
   hasMany,
+  Reference,
   hasOne,
   hasOneToOne,
-  Lens,
-  Loaded,
-  LoadHint,
-  loadLens,
-  newChangesProxy,
-  newRequiredRule,
-  OptsOf,
-  OrderBy,
-  PartialOrNull,
-  Reference,
   setField,
-  setOpts,
-  ValueFilter,
-  ValueGraphQLFilter,
 } from "joist-orm";
-import { Context } from "src/context";
 import {
   Author,
+  newAuthor,
   authorMeta,
+  Publisher,
+  Image,
   Book,
+  PublisherId,
+  ImageId,
   BookId,
+  PublisherOrder,
```

Comment:
request change: It kind of looks like you ran `yarn workspace format`, as requested in the `Readme`, but I think you should revert all these files under `entities`.

It would be nice if we could have these folders be ignored by the formatter, as I ran into that problem aslo.